docs/backups-storage.md

```diff
@@ -208,3 +208,65 @@ within the default memory limits.
             credentialsSecret: my-cluster-azure-secret
           ...
     ```
+
+## Remote file server
+
+Here is an example of the `deploy/cr.yaml` backup section fragment,
+which configures a private volume for filesystem-type storage:
+
+You can use `fileystem` backup storage type to mount a *remote file server* to
+a local directory as a *sidecar Volume*, and make Percona Backup for MongoDB
+using this directory as a storage for backus.
+
+The approach is based on using common
+[Network File System (NFS) protocol :octicons-link-external-16:](https://en.wikipedia.org/wiki/Network_File_System).
+Particularly, this storage type is useful in network-restricted environments
+without S3-compatible storage, or in cases with a non-standard storage service
+that still supports NFS access.
+
+1. Add the remote storage as a [sidecar volume](operator.md#replsetssidecarvolumesname)
+    in the `replset` section of the Custom Resource (and also in `configsvrReplSet`
+    in case of a sharded cluster). You will need to specify the server hostname
+    and some directory on it, as in the following example:
+
+    ```yaml
+    replsets:
+    - name: rs0
+      ...
+      sidecarVolumes:
+      - name: backup-nfs-vol
+        nfs:
+          server: "nfs-service.storage.svc.cluster.local"
+          path: "/psmdb-my-cluster-name-rs0"
+      ...
+    ```
+
+    The `backup-nfs-vol` name specified above will be used to refer this
+    sidecar Volume in the backup section.
```

Comment:
```suggestion
    sidecar volume in the backup section.
```

Reply:
Better we leave it as is, we capitalize k8s terms to make wording more clear